product_docs/docs/edbcloud/beta/getting_started/01_connect_cloud_account.mdx

```diff
@@ -43,27 +41,15 @@ To make requests to increase Azure Networking Limits follow [these instructions]
 
 To create an Azure Active Directory Application:
 1. Register an application with Azure AD and create a service principal using [these instructions](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#register-an-application-with-azure-ad-and-create-a-service-principal).
-Take note of the **Application (client) ID**, you need it at the final step. Copy **Display name** of Azure AD application for the use in the next step.
-You will need to enter **Display name** _as is_ in the **Select** field, at the bottom of  [**Add role assignment** panel](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current#step-2-open-the-add-role-assignment-pane)   the panel.
+Take note of the **Application (client) ID**, you need it to configure your cloud account. Also take note of the **Display name** of the Azure AD application. 
+You need to enter the **Display name** _as is_ in the next step of this procedure. 
 
-2. Assign _Owner_ role to the application as described [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application). Enter the **Display name** of the Azure AD application in the **Select** field of the **Add role assignment** panel.
+2. Assign _Owner_ role to the application as described [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#assign-a-role-to-the-application). Enter the **Display name** of the Azure AD application in the **Select** field of the [**Add role assignment** panel](https://docs.microsoft.com/en-us/azure/role-based-access-control/role-assignments-portal?tabs=current#step-2-open-the-add-role-assignment-pane) panel.
 
-3. Choose _application secret_ as an authentication option for the application. [Create a new Azure AD application secret. ](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret). After you created the secret, copy the value of Azure AD App Secret, you will need it for the final step.
+3. Choose _application secret_ as an authentication option for the application. [Create a new Azure AD application secret. ](https://docs.microsoft.com/en-us/azure/active-directory/develop/howto-create-service-principal-portal#option-2-create-a-new-application-secret) Take note of the Azure AD App Secret, you need it to configure your cloud account.
 
 4. If you can’t give away your existing Azure subscription, [create an additional Azure subscription](https://docs.microsoft.com/en-us/azure/cost-management-billing/manage/create-subscription).
```

Comment:
I don't understand what this statement means or why it is here

Reply:
The intent here is either if a user cannot use an existing subscription, then they will need to create a new subscription. Frankly, I would remove this altogether from here and state at the very beginning that a customer needs a Microsoft Azure subscription to get started.